### ISSUE TITLE
Magic plugin with strapi version 4 and higher

### ISSUE THREAD

**Issue by Jonny993** [GalloDaSballo/Next-Ecommerce-Backend#1]
Hi,
I tried to reach you on the youtube video as well but didn't have an answer back...

So I'm trying to follow along your tutorial on the latest strapi version as of today it is 4.0.0.

With the strapi plugin installed I should be able to see it on the Admin Panel but it doesn't show up.

Also, when trying to continue with the tutorial even doing the same exact things I couldn't authenticate through magic resulting in a 401 unhautorized...

I would be glad if you could guide me on this.

Thank you very much for your effort in putting this app.

**Comment by northrn** [comment]
Yep Magic plugin is broken on current version of Strapi. Not sure if there's any other way to integrate them.
